source/dbo/Functions/Power BI Reporting/ufnUkBorn.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE FUNCTION [dbo].[ufnUkBorn] (
+	@CountryId int
+)
+	RETURNS [nvarchar](10)
+AS
+	BEGIN
+		DECLARE @IsUkBorn nvarchar(10)
+		SET @IsUkBorn =
+		CASE 
+			WHEN @CountryId = 235 THEN 'Yes'
+			WHEN @CountryId = 238 THEN 'Unknown'
```

Comment:
Can the `CountryId` passed in here ever be NULL? If it can, would it be better to map NULL to Unknown?

Reply:
Country of birth is a mandatory field, whereas 'Unknown' is an actual value you can select through the front end.  So I think we would want that to output a null if it did happen, to make it easier to spot.

Reply:
Change made to output null if country id is null